.github/workflows/c_actions.yml

```diff
@@ -44,11 +44,6 @@ jobs:
         repository: SpiNNakerManchester/SupportScripts
         path: support
         
-    - name: Set GITHUB_BRANCH
```

Comment:
I would have thought GITHUB_BRANCH would be needed here, if there are branches with the same name that need to be checked out from spinn_common or spinnaker_tools?

Reply:
GITHUB_BRANCH was needed when we where doing pull requests as then the environment variable GITHUB_REF was not usable.

As we are only doing push ones clone2.sh directly uses GITHUB_REF so GITHUB_BRANCH is not longer needed to get any downstream branches with the same name

Reply:
Ah, okay - good point.  Needs removing in other C actions branches as well in that case... I'll do that.